Gems/Multiplayer/Code/Source/Editor/MultiplayerEditorSystemComponent.cpp

```diff
@@ -157,12 +160,21 @@ namespace Multiplayer
             [[fallthrough]];
         case eNotify_OnEndGameMode:
             // Kill the configured server if it's active
-            if (m_serverProcess)
+            if (m_serverProcessWatcher)
             {
-                m_serverProcess->TerminateProcess(0);
-                m_serverProcess = nullptr;
+                m_serverProcessWatcher->TerminateProcess(0);
+                if (m_serverProcessTracePrinter)
+                {
+                    m_serverProcessTracePrinter->Pump();
+                    m_serverProcessTracePrinter->WriteCurrentString(true);
+                    m_serverProcessTracePrinter->WriteCurrentString(false);
+                }
+                m_serverProcessWatcher = nullptr;
+                m_serverProcessTracePrinter = nullptr;
             }
 
+            AZ::TickBus::Handler::BusDisconnect();
```

Comment:
Detach and then kill the m_serverProcessWatcher? Otherwise you could still be ticking while you are pulling it down.

Reply:
Fixed!